settlement/local/local.go

```diff
@@ -54,6 +54,16 @@ type Client struct {
 	settlementKV store.KV
 }
 
+func (c *Client) GetStateInfo(index uint64) (*types.StateInfo, error) {
+	// TODO implement me
+	panic("implement me")
+}
+
+func (c *Client) GetRollapp() (*types.Rollapp, error) {
```

Comment:
i guess this will need to be implemented, otherwise it will not be possible to use mock settlement.